buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -1,5 +1,26 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "displayName": "run begin.sh",
+      "timeoutInMinutes": 0,
+      "alwaysRun": true,
+      "task": {
+        "id": "10f1f9a1-74b0-47ab-87bf-e3c9c68e8b0d",
+        "versionSpec": "0.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "type": "InlineScript",
+        "scriptPath": "",
+        "args": "",
+        "cwd": "",
+        "failOnStandardError": "false",
+        "script": "if [ -f \"$AGENTTOOLSPATH/begin.sh\" ]; then echo \"$AGENTTOOLSPATH/begin.sh script found. Executing...\"; $AGENTTOOLSPATH/begin.sh ; else echo \"$AGENTTOOLSPATH/begin.sh script does not exist. Moving on.\" ; fi"
```

Comment:
Why are you using $AGENTTOOLSPATH here Vs. $(AgentToolsPath)? You are defining the variable below as Pascal cased and you are using all caps when using the variable. Not pretty sure that actually works

Reply:
I had a similar thought but I believe this works I think VSTS set the env variables as all caps but it is worth verifying

Reply:
I had that doubt as well. I had checked that and double checked now, in Linux seems I need to use the all uppercase variant: https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build?_a=summary&buildId=1605959